src/convnets/mobilenets/mobilenetv3.jl

```diff
@@ -20,45 +16,57 @@ Create a MobileNetv3 model.
       + `r::Integer` - The reduction factor (`>= 1` or `nothing` to skip) for squeeze and excite layers
       + `s::Integer` - The stride of the convolutional kernel
       + `a` - The activation function used in the bottleneck (typically `hardswish` or `relu`)
-  - `inchannels`: The number of input channels.
+
+  - `width_mult`: Controls the number of output feature maps in each block
+    (with 1 being the default in the paper; this is usually a value between 0.1 and 1.4.)
   - `max_width`: The maximum number of feature maps in any layer of the network
+  - `dropout_rate`: The dropout rate to use in the classifier head. Set to `nothing` to disable.
+  - `inchannels`: The number of input channels.
   - `nclasses`: the number of output classes
 """
-function mobilenetv3(width_mult::Real, configs::AbstractVector{<:Tuple};
-                     max_width::Integer = 1024, inchannels::Integer = 3,
-                     nclasses::Integer = 1000)
+function mobilenetv3(configs::AbstractVector{<:Tuple}; width_mult::Real = 1,
```

Comment:
This is going to get updated now that the `SkipConnection`s are handled in `resnet` right?